src/server/test/routes/compareReadingsParamsTest.js

```diff
@@ -0,0 +1,321 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+const { expect } = require('chai');
+const { chai, mocha, app, testDB } = require('../common');
+const { validateString, testInvalidField } = require('../util/validationHelpers');
+
+mocha.describe('Compare Readings Parameter Validation', () => {
+
+    mocha.describe('GET /api/compareReadings/meters/:meter_ids - Meter Compare Readings', () => {
+        const BASE_METER_ENDPOINT = '/api/compareReadings/meters';
+        
+        const validQuery = {
+            curr_start: '2023-01-01T00:00:00.000Z',
+            curr_end: '2023-01-02T00:00:00.000Z',
+            shift: 'P1D',
+            graphicUnitId: '1'
+        };
+
+        mocha.describe('URL Parameter Validation - meter_ids', () => {
```

Comment:
I have not carefully checked but a number of these seems to be the type of tests that could apply across many routes with proper generalization. This was done for string, number, etc. Would it make sense to do that here?